llvm/test/CodeGen/AMDGPU/wwm-reg-shift-down-gfx11plus.mir

```diff
@@ -0,0 +1,30 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
+# RUN: llc -mtriple=amdgcn -mcpu=gfx1100 -mattr=+real-true16 -run-pass=prologepilog %s -o - | FileCheck -check-prefix=GCN %s
+
+---
+name:            wwm_skip_shift_16bit_reg
+tracksRegLiveness: true
+noVRegs:           true
+machineFunctionInfo:
+  wwmReservedRegs: ['$vgpr0_lo16']
+  isEntryFunction: false
+body:             |
+  bb.0:
+    liveins: $sgpr0, $sgpr1
+    ; GCN-LABEL: name: wwm_skip_shift_16bit_reg
+    ; GCN: liveins: $sgpr0, $sgpr1
+    ; GCN-NEXT: {{  $}}
+    ; GCN-NEXT: $sgpr2 = S_XOR_SAVEEXEC_B32 -1, implicit-def $exec, implicit-def dead $scc, implicit $exec
+    ; GCN-NEXT: SCRATCH_STORE_DWORD_SADDR killed $vgpr1, $sp_reg, 0, 0, implicit $exec, implicit $flat_scr :: (store (s32) into %stack.0, addrspace 5)
+    ; GCN-NEXT: $exec_lo = S_MOV_B32 killed $sgpr2
+    ; GCN-NEXT: renamable $vgpr0_hi16 = IMPLICIT_DEF
+    ; GCN-NEXT: $sgpr4 = ENTER_STRICT_WWM -1, implicit-def $exec, implicit-def $scc, implicit $exec
+    ; GCN-NEXT: $vgpr1 = IMPLICIT_DEF
+    ; GCN-NEXT: $vgpr1 = V_CNDMASK_B16_t16_e64 0, killed $vgpr0_hi16, 0, $vgpr1, $sgpr0, 0, implicit $exec
```

Comment:
This is wrong. The original register, vgpr0_lo16, has been replaced with vgpr1. The V_CNDMASK_b16_t16_e64 inst is supposed to define a 16-bit register, isn't it? There is a bug.